apps/console/src/features/identity-providers/components/forms/authenticators/google-authenticator-form.tsx

```diff
@@ -435,6 +462,31 @@ export const GoogleAuthenticatorForm: FunctionComponent<GoogleAuthenticatorFormP
                 width={ 16 }
                 data-testid={ `${ testId }-additional-query-parameters` }
             />
+            { isSuperTenantLogin() 
+                ? (
+                    <Field.Checkbox
+                        ariaLabel="Enable Google One Tap as a sign in option"
+                        name="IsGoogleOneTapEnabled"
+                        required={ false }
+                        requiredErrorMessage={ "Please select" }
```

Comment:
There should be a common required error message in `console:develop.features.authenticationProvider.forms.authenticatorSettings`.

Reply:
Fixed 23103f54f995af0472e650905da7201bceda5769
Removed the requiredErrorMessage attribute since it does not make sense for a toggle